cli/config/compose/services/kibana/docker-compose.yml

```diff
@@ -5,6 +5,6 @@ services:
       - ELASTICSEARCH_HOSTS=http://elasticsearch:9200
       - ELASTIC_USERNAME=elastic
       - ELASTIC_PASSWORD=changeme
-    image: "docker.elastic.co/observability-ci/kibana:${kibanaTag}"
```

Comment:
@mdelapenya can you discuss the differences in build times and availability of images between the observability-ci and main elastic docker images?   Will this change prevent us from 'getting' a fix into the test environment faster, presuming the main docker image is built only once or twice a day, and the obs-ci is presumably built much more often?  

I was also thinking in a separate issue we should dry this up and up put the version portion of the semVar in a constant.

Reply:
The observability-ci namespace is refreshed with every merge to master in beats, and for the stack they are updatedd automatically.

So our elastic-agent and metricbeat images and binaries will be always synced with the branch